src/providers/LoggerProvider/index.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useContext } from 'react';
+
+import { ConsoleLogger, Logger, LogLevel } from 'amazon-chime-sdk-js';
+
+export const LoggerContext = React.createContext<Logger|undefined>(undefined);
+
+interface Props {
+  /**
+   * Represents [LogLevel](https://aws.github.io/amazon-chime-sdk-js/enums/loglevel.html) to determine how verbose the logging should be.
+   */
+  logLevel?: LogLevel;
+  
+  /**
+   * This name will be used as a prefix when we log statements.
+   */
+  name?: string;
```

Comment:
I thought we would do multiLogger. I think that is important since that is how customer store logs and send to us right?

Reply:
So we have two routes:
1. Get POSTLogger config but for that builders must pass `MeetingSessionConfiguration` as well and then only we can create a `MultiLogger`.
2. Provide a `Logger` prop and let builders pass any class implementing `Logger` interface, but if not provided we fall back to `ConsoleLogger` which itself can be customize with a `name` and `LogLevel`.

I have went with option 2 as anyhow I think we would have to provide the `Logger` prop to let builders pass their own prop. I have provided a usage example as well on how to provide a custom logger.

Reply:
So in the case of multiLogger where they post logs to CloudWatch, how would they have both React and meeting post to the same log group then? Will they have to create one Logger and pass to post the `MeetingProvider` and `LoggerProvider`?
You will need to do that with the react meeting demo right?

Reply:
Yes, they have to create and pass to both the MeetingProvider and LoggerProvider. We want to make `LoggerProvider `completely independent from any meeting specific configuration hence we discussed to only support `ConsoleLogger` initially as constructing the POSTLogger needs the meeting session configuration.

Reply:
The POST logging IMO can be completely dropped until we support creating the `MeetingSessionConfiguration` before calling the `meetingManager.join()` then only we could log to same group. Also, the POST logger taking the meetingSessionConfiguration will still have the same attendeeId and meetingId as it would be created from the same CreateAttendee and CreateMeeting responses but it is not a good way I think.